easybuild/easyconfigs/n/NVHPC/NVHPC-22.9-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,76 @@
+name = 'NVHPC'
+version = '22.9'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+local_tarball_tmpl = 'nvhpc_2022_%%(version_major)s%%(version_minor)s_Linux_%s_cuda_multi.tar.gz'
+# By downloading, you accept the HPC SDK Software License Agreement
+# https://docs.nvidia.com/hpc-sdk/eula/index.html
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = [
+    {
+        local_tarball_tmpl % 'aarch64':
+            'bc4473f04b49bc9a26f08c17a72360650ddf48a3b6eefacdc525d79c8d730f30',
+        local_tarball_tmpl % 'ppc64le':
+            '9aac31d36bb09f6653544978021f5b78c272112e7748871566f7e930f5e7475b',
+        local_tarball_tmpl % 'x86_64':
+            'aebfeb826ace3dabf9699f72390ca0340f8789a8ef6fe4032e3c7b794f073ea3',
+    }
+]
+
+local_gccver = '11.3.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.38', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.7.0', '', SYSTEM),
+]
+
+module_add_cuda = False
+
+# specify default CUDA version that should be used by NVHPC
+# should match one of the CUDA versions that are included with this NVHPC version
+# (see install_components/Linux_x86_64/22.3/cuda/)
+# for NVHPC 22.3, those are: 11.6, 11.0, 10.2;
```

Comment:
```suggestion
# (see install_components/Linux_x86_64/ver/cuda/) where ver is the NVHPC version
```

The suggestion makes this comment more generic and avoids the need to update it for each version. The alternative is that this is changed to be correct for 22.9.

Similarly for 22.11 in #17228 and 23.1 in #17229